tests/unit-core/test-api-set-and-clear-error-flag.c

```diff
@@ -31,6 +31,82 @@ main (void)
 
   JERRY_ASSERT (obj_val != err_val);
   jerry_release_value (err_val);
+  jerry_release_value (obj_val);
+
+  jerry_value_t str = jerry_create_string ((jerry_char_t *) "Pterodactylus");
+  jerry_value_t error = jerry_create_error_from_value (str, true);
+  str = jerry_get_value_from_error (error, true);
+  jerry_size_t str_size = jerry_get_string_size (str);
+  jerry_char_t str_buff[str_size];
+  jerry_string_to_utf8_char_buffer (str, str_buff, str_size);
+  str_buff[str_size] = '\0';
+  JERRY_ASSERT (!strcmp ("Pterodactylus", (char *) str_buff));
+  jerry_release_value (str);
```

Comment:
This part from `jerry_size_t str_size = jerry_get_string_size (str);` repeated many times. Perhaps we could introduce a helper function for it. Something like a `void compare_str_to(ecma_value_t value, const char *str_p, size_t str_len)`